compliance_checker/tests/data/non-comp/bad.cdl

```diff
@@ -45,10 +45,7 @@ variables:
 		lev2:standard_name = "atmosphere_hybrid_height_coordinate" ;
 		lev2:formula_terms = "a: var1 b: var2 orog: var3" ;
 		lev2:description = "Missing variables from formula_terms" ;
-	double bad_time_1(time) ;
-		bad_time_1:standard_name = "time" ;
-		bad_time_1:description = "lacks units" ;
-	double bad_time_2(time) ;
+	double time(time) ;
 		bad_time_2:units = "seconds" ;
```

Comment:
Ooops, thought I had changed over the prefixes.  I need to change the variable name and/or the dimension name, as the `bad_time_1(time)` is not a coordinate variable as the var name isn't the same as the coordinate name.